diff --git a/node_modules/react-native-scoped-storage/android/build.gradle b/node_modules/react-native-scoped-storage/android/build.gradle
index 9a59b70..fdc5fba 100644
--- a/node_modules/react-native-scoped-storage/android/build.gradle
+++ b/node_modules/react-native-scoped-storage/android/build.gradle
@@ -2,20 +2,6 @@ def safeExtGet(prop, fallback) {
     rootProject.ext.has(prop) ? rootProject.ext.get(prop) : fallback
 }
 
-buildscript {
-  repositories {
-    jcenter()
-    google()
-    maven {
-      url 'https://maven.google.com'
-    }
-  }
-
-  dependencies {
-    classpath 'com.android.tools.build:gradle:3.4.2'
-  }
-}
-
 apply plugin: 'com.android.library'
 
 android {
